tests/integration/test_sqlalchemy_integration.py

```diff
@@ -58,6 +58,8 @@ def test_has_table(self, engine: Engine, fact_table_name: str):
     def test_get_table_names(self, engine: Engine):
         results = engine.dialect.get_table_names(engine)
         assert len(results) > 0
+        results = engine.dialect.get_table_names(engine, "public")
+        assert len(results) > 0
```

Comment:
Can we also add a test for empty results?

Reply:
good idea, added.